### ISSUE TITLE
Get Member.inviter as an addition to the functionality of member.

### ISSUE THREAD

**Issue by MySixSenses** [Rapptz/discord.py#5204]
### The Problem

I would like a feature that adds to the functionality of Member by adding an attribute to member which shows the last member to invite the Member.
### The Ideal Solution
```
print("str(Member.inviter)") #returns ExampleUser#0001
```

### The Current Solution
```
import asyncio
import datetime
import json
import os
import commands

client = discord.Client()
cfg = open("config.json", "r")
tmpconfig = cfg.read()
cfg.close()
config = json.loads(tmpconfig)

token = config["token"]
guild_id = config["server-id"]
logs_channel = config["logs-channel-id"]


invites = {}
last = ""

async def fetch():
 global last
 global invites
 await client.wait_until_ready()
 gld = client.get_guild(int(guild_id))
 logs = client.get_channel(int(logs_channel))
 while True:
  invs = await gld.invites()
  tmp = []
  for i in invs:
   for s in invites:
    if s[0] == i.code:
     if int(i.uses) > s[1]:
      usr = gld.get_member(int(last))
      testh = f"{usr.name} **joined**; Invited by **{i.inviter.name}** (**{str(i.uses)}** invites)"
      await logs.send(testh)
   tmp.append(tuple((i.code, i.uses)))
  invites = tmp
  await asyncio.sleep(4)


@client.event
async def on_ready():
 print("ready!")



@client.event
async def on_member_join(meme):
 global last
 last = str(meme.id)




client.loop.create_task(fetch())
client.run(token)
```

### Summary

TL;DR:
New feature to member called inviter which shows the last member to invite someone in a guild.

**Comment by Vexs** [comment]
This isn't something that can be reasonably provided by this library. It isn't provided by discord, and requires long-term permanent storage to do, which falls *completely* out of scope.

**Comment by MySixSenses** [comment]
Closed, see ^
